tests/python/ConfigTest.py

```diff
@@ -773,83 +775,287 @@ def test_virtual_display(self):
+                         self.cfg.getVirtualDisplayViewColorSpaceName(view_name))
+        self.assertEqual('', self.cfg.getVirtualDisplayViewLooks(view_name))
+        self.assertEqual('', self.cfg.getVirtualDisplayViewRule(view_name))
+        self.assertEqual(
+            '', self.cfg.getVirtualDisplayViewDescription(view_name))
+
+    def test_get_virtual_display_views_shared(self):
+        views = self.cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)
+        self.assertEqual(views.__len__(), 1)
+        view_name = views[0]
+        self.assertEqual(view_name, 'sview2')
+
+    def test_remove_view_from_virtual_display(self):
+        """
+        Test remove a view from the Virtual Display.
+        """
+
+        self.cfg.removeVirtualDisplayView('Raw')
+        views = self.cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)
+        self.assertEqual(views.__len__(), 1)
+        view_name = views[0]
+        self.assertEqual(view_name, 'Film')
+
+        views = self.cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)
+        self.assertEqual(views.__len__(), 1)
+        view_name = views[0]
+        self.assertEqual(view_name, 'sview2')
+
+        # Test remove a shared view from the Virtual Display.
+
+        self.cfg.removeVirtualDisplayView('sview2')
+        views = self.cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)
+        self.assertEqual(views.__len__(), 1)
+        views = self.cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)
+        self.assertEqual(views.__len__(), 0)
+
+        # Extra serialize & deserialize validation.
+        cfg = OCIO.Config().CreateFromStream(self.cfg.serialize())
+        views = cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)
+        self.assertEqual(views.__len__(), 1)
+        views = cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)
+        self.assertEqual(views.__len__(), 0)
+
+        self.cfg.addVirtualDisplaySharedView('sview2')
+        views = self.cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)
+        self.assertEqual(views.__len__(), 1)
+        views = self.cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)
+        self.assertEqual(views.__len__(), 1)
+
+    def test_remove_virtual_display(self):
+        """
+        Test remove the Virtual Display.
+        """
+
+        self.cfg.clearVirtualDisplay()
+        views = self.cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)
+        self.assertEqual(views.__len__(), 0)
+        views = self.cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)
+        self.assertEqual(views.__len__(), 0)
+
+        # Extra serialize & deserialize validation.
+        cfg = OCIO.Config().CreateFromStream(self.cfg.serialize())
+        views = cfg.getVirtualDisplayViews(OCIO.VIEW_DISPLAY_DEFINED)
+        self.assertEqual(views.__len__(), 0)
+        views = cfg.getVirtualDisplayViews(OCIO.VIEW_SHARED)
+        self.assertEqual(views.__len__(), 0)
+
+    def test_virtual_display_with_active_displays(self):
```

Comment:
The code now switches to another config file which is confusing because the previous methods were all related to another config file. To make it clear that it's now a new config to test, could you create another test case?